catala/sortir/sortir.md

```diff
@@ -5,4 +5,5 @@ Cada cop estàs més enfonsat... de sobte mires a d'alt i veus
 uns peus que es belluguen amb impaciència... aquestes sabates 
 et sonen... Ets tu!
 Tanques els ulls de nou, amb força, i dessitges que tot el que 
-has vist sigui fruit de la teva imaginació.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.